UI.tsv

```diff
@@ -325,7 +325,7 @@ UI_20150317_000324	{@st56}Dead{/}
 UI_20150317_000325	Spell Power
 UI_20150317_000326	Defense
 UI_20150317_000327	Magic Resistance
-UI_20150317_000328	Critical Rate
+UI_20150317_000328	Critical Attack Chance
```

Comment:
'Critical Rate' is fine, but its a game language understood only by gamers. Also, the meaning, in literal sense, does not mean 'your chance of landing a critical attack'. Of course, people who plays games, will understand easily.
'Critical Attack Chance' is more suited for both gamers and new players who are playing for the first time, more clear without any doubt of meaning. I want this game to be more accommodating even to people who never played an MMO or games in general.